.github/workflows/ci.yml

```diff
@@ -189,6 +189,13 @@ jobs:
     - run: nox -e ${{ matrix.task.nox }}
       if: ${{ matrix.task.run-if }}
 
+  package:
+    name: Build & verify package
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: hynek/build-and-inspect-python-package@v1
```

Comment:
Should we pin this to a specific commit?

Similar to that we did for

```
uses: pypa/gh-action-pypi-publish@37f50c210e3d2f9450da2cd423303d6a14a6e29f
```

This is mainly for security reasons.

Reply:
is someone keeping those pins up to date?

Reply:
I think that GHA dependencies should be handled as any other testing / build / CI dependency.

And as a general rule, we should pin every package that we use during the build system

Now, another question.

Do "hynek/build-and-inspect-python-package" need to be a GHA action ?
Can it be distributed installed and executed just like any other python package?

Just asking.

Reply:
That would be pointless since it doesn't do much novel things except calling established tools. This way you add only one like of YAML and it will just work, including handling package dependencies.